### ISSUE TITLE
Q6.2 and Q6.2a

### ISSUE THREAD

**Issue by alanjleonard** [globalmangrovewatch/gmw-users#78]

**Comment by julia-rom** [comment]
similar to grouped checkbox items but with a start end date. This will have to be a custom component (since we are not using the 'other' or 'name' field. My estimate is 3-5 days.

**Comment by mmnoo** [comment]
Agree, but adding a 2x buffer since stuff always happens. 3-10 days.  I have modified group checkbox to accommodate nested question per option, but its still WIP.

**Comment by mmnoo** [comment]
Caveat, my nested question modifications dont apply to just some checkboxes. They would be the same for all checkboxes in a group.
